lucene/core/src/java/org/apache/lucene/search/Scorable.java

```diff
@@ -30,6 +30,13 @@
    * Returns the score of the current document matching the query.
    */
   public abstract float score() throws IOException;
+  
+  /**
+   * Returns the smoothing score of the current document matching the query.
+   * This score is used when the query/term does not appear in the document.
+   * This can return 0 or a smoothing score.
```

Comment:
Maybe link to Indri paper that describes/motivates this?

Reply:
I have added more detail and a paper the describes the motivation of the smoothing score.  The description of how the smoothing score is used is at the bottom of page 11 in the paper.  It is important to note that most of the explanation has to do with when the score is a product.  Even though the IndriAndScorer does not use a product, the smoothing score is still helpful for acting like an idf.  Additionally, there are many more Indri operators that I would like to add that do use a product.

Reply:
Great, thanks.